visualization/app/codeCharta/state/store/fileSettings/attributeDescriptors/attributeDescriptors.action.ts

```diff
@@ -0,0 +1,31 @@
+import { AttributeDescriptors, CCAction } from "../../../../codeCharta.model"
+
+export enum AttributeDescriptorsActions {
+	SET_ATTRIBUTE_DESCRIPTORS = "SET_ATTRIBUTE_DESCRIPTORS"
+}
+
+export interface SetAttributeDescriptorsAction extends CCAction {
+	type: AttributeDescriptorsActions.SET_ATTRIBUTE_DESCRIPTORS
+	payload: AttributeDescriptors
+}
+
+export type AttributeDescriptorsAction = SetAttributeDescriptorsAction
+
+export function setAttributeDescriptors(
+	attributeDescriptors: AttributeDescriptors = defaultAttributeDescriptors
+): SetAttributeDescriptorsAction {
+	return {
+		type: AttributeDescriptorsActions.SET_ATTRIBUTE_DESCRIPTORS,
+		payload: attributeDescriptors
+	}
+}
+
+export const defaultAttributeDescriptors: AttributeDescriptors = {
+	DefaultAttributeDescriptor: {
```

Comment:
I would expect an empty list of descriptors rather a default one if we do not have any.